### PR TITLE
removed warning for depriciated activeCount by replacing it with acti…

### PR DESCRIPTION
**Pull request by aalekh-ul**
…ve_count and added a function which can print the output even when filename is given.

### CODE REVIEW

File: tor-map.py [hunk 1]
```diff
@@ -280,13 +285,18 @@ def main():
...
 
     ## Checking for file output
+
+    
+
     if OUTFILE != "empty_outfile":
         f = open(OUTFILE, "w")
-    else:
-        f = sys.stdout
+    def writeFile(text):
```
Comment (nishitm) [inline tor-map.py:293]: what is the need for this? can you explain?

Comment (aalekh-ul) [reply tor-map.py:293]: user can see the output while script is running.

Comment (nishitm) [reply tor-map.py:293]: ok, this is fine

**Review by nishitm — APPROVED** [review]
thanks for fixing it

**Pull request merged**